### PR TITLE
Document the new update process

### PR DESCRIPTION
**Pull request by zero-24**
Thanks @tianon 
PS do you still need access to the https://github.com/joomla-docker/official-images repo with your peronal accout? I guess only the @docker-library-bot need access now right?

cc @HLeithner

### CODE REVIEW

**Comment by tianon** [comment]
> Thanks @tianon
> PS do you still need access to the https://github.com/joomla-docker/official-images repo with your peronal accout? I guess only the @docker-library-bot need access now right?

My pleasure!  Yes, you can feel free to remove me from the repo now. :+1:

**Comment by zero-24** [comment]
Done thanks for your help on this 👍

File: README.md [hunk 1]
```diff
@@ -12,3 +12,11 @@ See a change merged here that doesn't show up on the Docker Hub yet? Check [the
...
+
+# How to update the official docker-library/official-images repo
+
+- [Since this Discussion with docker](https://github.com/docker-library/official-images/pull/8842#issuecomment-705099610) the @docker-library-bot has access to the [joomla-docker/official-images](https://github.com/joomla-docker/official-images)
+- That bot [runs every six hours](https://github.com/docker-library/official-images/pull/8842#issuecomment-705099610) over the changes from this repo and prepares an branch to update the offical repo.
+- So after a PR is merged here someone with write access to [joomla-docker/official-images](https://github.com/joomla-docker/official-images) can send a PR against the official repo.
+- Such a user can use this link: https://github.com/docker-library/official-images/compare/master...joomla-docker:joomla that should already include all the changes required as well as the changelog as generated by the @docker-library-bot
+- Please us that link to open the PR against the main repo [docker-library/official-images](https://github.com/docker-library/official-images).
```
Comment (wilsonge) [inline README.md:22]:
```suggestion
- Please use that link to open the PR against the main repo [docker-library/official-images](https://github.com/docker-library/official-images).
```

**Review by wilsonge — APPROVED** [review]

**Comment by wilsonge** [comment]
Nice work guys. Thankyou!

**Comment by zero-24** [comment]
@HLeithner please merge here when ok

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**